scripts/sync-up-helm-chart.sh

```diff
@@ -32,7 +32,37 @@ if [ -n "${crd_diff}" ]; then
   cp -f $INSTALL_CRDS_YAML $CHART_FEDERATED_CRD_DIR/crds.yaml
 fi
 
+# Start kube-apiserver to generate consistent CRD validations
+./bin/etcd > /dev/null 2>&1 &
+sleep 5
```

Comment:
Instead of using `sleep`, how about check if we can use `util::wait-for-condition` at https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/util.sh#L30 ?

Reply:
good util in waiting for condition. Addressed